test/parallel/test-fs-symlink.js

```diff
@@ -56,6 +56,10 @@ fs.symlink(linkData, linkPath, common.mustCall(function(err) {
     assert.ifError(err);
     assert.strictEqual(destination, linkData);
   }));
+
+  tmpdir.refresh();
```

Comment:
This seems like a probable bug. Synchronously removing and recreating the tmpdir in an asynchronous part of the test while other asynchronous parts of the test are using it seems like a race condition.

Reply:
Oh, I see, everything else in the test uses invalid stuff... so maybe it's OK. I'd still prefer it isolated to its own test file or something. Someone can add something to this test later and this would be easy to overlook as a likely problem.

Reply:
Addressed in https://github.com/nodejs/node/pull/34569